interop_client/src/main.rs

```diff
@@ -43,6 +45,22 @@ use tracing::{debug, error, info, instrument, trace, Span};
 use tracing_subscriber::EnvFilter;
 
 const IMPLEMENTATION_NAME: &str = "OpenMLS";
+const PROPOSAL_TYPES: [ProposalType; 6] = [
```

Comment:
This shouldn't be necessary here.
These are all default types and are assumed to be present.

Reply:
That was a change because of my PR for GCE support. As members are added, they are checked if they have all the required capabilities from the group. If they don't the add proposal would fail... For this PR itself this is not relevant...

Reply:
These are all default capabilities that do not need to be added. They should be added when used, even though the spec isn't entirely clear here. But they are not needed in general.